Capstone Project: Peruvian Software Conference 2023

paulsaenzsucre opened the pull request:
In this PR we:

📌 Adapt a Behance design for a fictitious conference in my country. This is the final assignmentt of module one named Capstone Project.
📌 Add code to dynamically create speaker's section.
📌 Add code to read a JSON file with the data.
📌 Adapt the index and about pages to desktop.
📌 Add HTML CSS and JavaScript to implement a design.

paulsaenzsucre committed:
Create linters.yml

Create the linter actions.

paulsaenzsucre committed:
- Setup the repository and tools.
- Add the basic HTML, CSS and Javascript project structure.

paulsaenzsucre committed:
Add images icons and fonts.

paulsaenzsucre committed:
Add home page.

paulsaenzsucre committed:
Add about page.

paulsaenzsucre committed:
Adapt the two pages to desktop.

paulsaenzsucre committed:
Adding code  dinamically create speakers section.

paulsaenzsucre committed:
Fix linter errors.

BertrandConxy commented on README.md:
```diff
@@ -1,2 +1,316 @@
-# PSC2023-Capstone
- This repository is for an fictisious conference on my country. This is the final assigment of module 1 named Capstone Project.
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800&display=swap');
+
+  /*Import box icons*/
+  @import url('https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css');
+
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+</style>
```
- [ ] Since these codes are not playing important role in the README, I would suggest that you remove them 😃 this will make your README look good 👍

BertrandConxy commented on README.md:
```diff
@@ -1,2 +1,316 @@
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+<!--
+  <h3><b>Microverse README Template</b></h3>
```
- [ ] From the requirements given, We should remove the Microverse logo in our README and if we have our own logo, we can replace it with it. Please remove it 😃 👍

BertrandConxy commented on README.md:
```diff
@@ -1,2 +1,316 @@
+
+To see a Live Demo Version of our page, you kindly follow the next link:
+
+- [Live Demo Link](https://elafildecolor.github.io/PSC2023-Capstone/)
```
- [ ] Good job on your README, it looks great!! However, this Live Demo link is not working. Please work on it again to make sure that it is working 😃 👍

BertrandConxy commented on README.md:
```diff
@@ -1,2 +1,316 @@
+- Twitter: [@paulsaenzsucre](https://twitter.com/paulsaenzsucre) <i class='bx bxl-twitter bx-tada' style='color:#44d7ff' ></i>
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/paulsaenzsucre) <i class='bx bxl-linkedin bx-tada' style='color:#278ace'  ></i>
+<!--
+<i class='bx bxs-user-badge bx-tada' style='color:#ace1f3' ></i> 
+
+
+**Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle) <i class='bx bxl-github bx-tada' style='color:#efefef' ></i>
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle) <i class='bx bxl-twitter bx-tada' style='color:#44d7ff' ></i>
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) <i class='bx bxl-linkedin bx-tada' style='color:#278ace'  ></i>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+-->
+<!-- FUTURE FEATURES -->
```
- [ ] I think it would be good if you could remove all the content that is commented on in your README file. It does not appear on the render but It makes your README look good 😃 😎

BertrandConxy commented on index.html:
```diff
@@ -0,0 +1,105 @@
+          <p class="prog-card-desc">It allows users to find events, communicate with people attending events such as conferences, trade fairs and other events. It allows users to search for people with similar interests.</p>
+        </div>
+      </div>
+      
+      <button id="join-btn" class="join-btn" type="button">Join PS Conference 2023</button>
+      <a class="hidden" href="">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="partner-cont" class="partner-cont">
```
- [ ] Good job 👏 However, from the design guidelines there is a difference between your design and the design given. Please look at the pictures below 👇 
**Given Design**
![image](https://user-images.githubusercontent.com/90222110/204053668-65ce748a-5be3-4335-be36-423d53be4ca3.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/204053726-0b57cf43-3882-49cb-8428-edd6fd742787.png)

But you should keep in mind that you will maintain the design for the mobile screen 😎

BertrandConxy commented on about.html:
```diff
@@ -0,0 +1,89 @@
+        <a class="partner-logo" href=""><img src="images/naver.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/daumkakao.svg" alt=""></a>
+        <a class="partner-logo" href=""><img src="images/airbnb.svg" alt=""></a>
+      </div>
+    </section>
+    <footer class="footer-cont flex-cont">
+      <img class="footer-logo" src="images/logo-ps.svg" alt="">
+      <div class="footer-desc">
+        <p>2023 Paracas<span class="orange">Soft</span>.</p>
+        <p>Some Rights Reserved.</p>
+      </div>      
+    </footer>
+  </main>
```
- [ ] Please note that the footer design for the `about page` should look dark which is different for your design. Please look at the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/204054025-9fef76ad-881c-400f-9394-4995b1b9c45f.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/204054075-d32a7f1a-5563-4a94-a275-ef8b0b333ecf.png)

BertrandConxy reviewed (changes requested):
Hi @elafildecolor, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR 💯 
- Good video presentation 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

paulsaenzsucre committed:
- Fix the REAME file.
- Add the more button funcionality.
- Fix the about page footer.
- Fix the desktop link on destop version.

paulsaenzsucre committed:
Fix linter errors.

paulsaenzsucre committed:
Fix the relative path of backgroun images

paulsaenzsucre committed:
Fix relative path of links in CSS

Favourezeugwa reviewed (approved):
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @elafildecolor , 

I am Favour 👩🏽 , your reviewer for this capstone.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

![happy](https://user-images.githubusercontent.com/99246359/202491301-53f8c6a2-8ff4-4337-9b6b-9c75c8bdf345.gif)

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
